jhipster-jwt-backend/src/main/java/com/okta/developer/web/rest/AccountResource.java

```diff
@@ -65,6 +65,34 @@ public void registerAccount(@Valid @RequestBody ManagedUserVM managedUserVM) {
         mailService.sendActivationEmail(user);
     }
 
+
+    /**
+       * POST  /register : register the user.
+       *
+       * @param managedUserVM the managed user View Model
+       * @throws InvalidPasswordException 400 (Bad Request) if the password is incorrect
+       * @throws EmailAlreadyUsedException 400 (Bad Request) if the email is already used
+       * @throws LoginAlreadyUsedException 400 (Bad Request) if the login is already used
+       */
+       @PostMapping("/registerFacebookMobile")
+       @Timed
+       @ResponseStatus(HttpStatus.CREATED)
+       public void registerAccountFacebook(@Valid @RequestBody ManagedUserVM managedUserVM) {
```

Comment:
I was trying to understand what is happening here, as I am trying to achieve the same, that is to signin/signup user from Ionic frontend with social login with Jhipster backend. 
But I was under assumption that, we should be getting some token here received from Facebook native app, and then Jhipster app should again be verifying that token with Facebook.

Kindly help me to understand the flow, how did we directly getting ManagedUserVM here? What if someone directly call /registerFacebookMobile with ManagedUserVM object?

Reply:
Hi! 
ok, this login/signup implementation works with the [ionic Facebook plugin](https://ionicframework.com/docs/native/facebook/) , which is the one in charge of verify the user and authorize your app, after doing this,  you can call the Facebook api from your ionic app  and get the ManagedUserVM to be send  to the jhipster backend.

I did this implementation because I couldn't  put to work the social components of Jhipster in my ionic app at that moment, But it was straight forward with the Ionic plugin. 

But I think if you want to try to implement the jhi-social component, you need to change the redirection of the signUp method in SocialController.java... buff anyways, this is a different approach from the one that jhipster already has, but please let me know if you manage to implement the jhipster social flow in your ionic app !